src/index.ts

```diff
@@ -174,17 +217,36 @@ class SvgChunkWebpackPlugin {
 	 * @param {String} entryName Entrypoint name
 	 * @returns {Array<Object>} Svgs list
 	 */
-	getSvgsDependenciesByEntrypoint(entryName: string): Array<NormalModule> {
+	getSvgsDependenciesByEntrypoint({
+		compilation,
+		entryName
+	}: {
+		compilation: any;
+		entryName: string;
+	}): Array<NormalModule> {
 		const listSvgsDependencies: Array<NormalModule> = [];
 
-		this.compilation.entrypoints.get(entryName).chunks.forEach((chunk: Chunk) => {
-			for (const module of this.compilation.chunkGraph.getChunkModulesIterable(chunk)) {
+		compilation.entrypoints.get(entryName).chunks.forEach((chunk: Chunk) => {
+			for (const module of compilation.chunkGraph.getChunkModulesIterable(chunk)) {
 				if (module.buildInfo && module.buildInfo.SVG_CHUNK_WEBPACK_PLUGIN) {
 					listSvgsDependencies.push(module);
 				}
 			}
 		});
 
+		// Need to sort (**always**), to have deterministic build
+		listSvgsDependencies.sort((a: NormalModule, b: NormalModule) => {
+			if (a.name < b.name) {
```

Comment:
@alexander-akait The `name` property exist on a `NormalModule`? I didn't find reference about it

Reply:
Yeah, my mistake, I will fix it, looks like we don't have properly type here